src/components/interfaces/HMI_API.xml

```diff
@@ -584,12 +584,12 @@
   <element name="showConstantTBTNextTurnIcon">
     <description>The secondary image field for ShowConstantTBT</description>
   </element>
-  <element name="audiPassThruIcon">
-    <description>The optional image for AudioPassThru</description>
-  </element>
   <element name="locationImage">
     <description>The optional image of a destination / location</description>
   </element>
+  <element name="audiPassThruIcon">
```

Comment:
@LuxoftAKutsan done in https://github.com/LuxoftAKutsan/sdl_core/pull/113/commits/ab9adc7c6996e68e910941d6e3ed04f751e49215